gremlin-dotnet/src/Gremlin.Net/Driver/ConnectionPoolSettings.cs

```diff
@@ -131,5 +133,19 @@ public TimeSpan ReconnectionBaseDelay
                 _reconnectionBaseDelay = value;
             }
         }
+        
+        /// <summary>
+        /// Gets or sets whether a connection pool will send a user agent during web socket handshakes
+        /// </summary>
+        /// <remarks>
+        /// The default value is true. When enabled, user agents will only be sent during the web socket
+        /// handshake. User agent follows the form:
+        /// [Application Name] [GLV Name].[Version] [Language Runtime Version] [OS].[Version] [CPU Architecture]
+        /// </remarks>
+        public bool EnableUserAgentOnConnect
+        {
+            get => _enableUserAgentOnConnect;
+            set => _enableUserAgentOnConnect = value;
+        }
```

Comment:
No need to introduce field
```suggestion
        public bool EnableUserAgentOnConnect  { get;  set; } = DefaultEnableUserAgentOnConnect;
```

Reply:
Thanks that looks a lot cleaner